api/views/student.py

```diff
@@ -141,6 +177,24 @@ def available_students_list(self, request: Request) -> Response:  # noqa: ARG002
             status=status.HTTP_200_OK,
         )
 
+    @extend_schema(
+        responses={
+            status.HTTP_200_OK: OpenApiResponse(
+                response=ListSerializer(child=DashboardStudentSerializer()),
+                description="Students returned",
+            ),
+        }
+    )
+    @action(detail=False, methods=["get"])
+    def active_students_with_no_groups(self, request: Request) -> Response:  # noqa: ARG002
+        matched_students = Student.objects.filter(
+            project_status=StudentProjectStatus.STUDYING
```

Comment:
Don't we need any additional filter by situational status here? @lemontree210

Reply:
Thanks for checking - no, I don't think so